.changeset/four-fishes-applaud.md

```diff
@@ -0,0 +1,6 @@
+---
+'@leafygreen-ui/menu': patch
```

Comment:
nit rm?

Reply:
I think it's still a valid changeset. The script will merge the two comments into one